recipe/meta.yaml

```diff
@@ -74,15 +100,24 @@ requirements:
     - jpeg 9*
     - libtiff 4.0.*
     - zlib 1.2.*
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
     - icu 58.*
+    - xorg-libice      # [linux]
+    - xorg-libsm       # [linux]
+    - xorg-libx11      # [linux]
+    - xorg-libxext     # [linux]
+    - xorg-libxrender  # [linux]
+    - xorg-libxt       # [linux]
+    - sqlite
 
 test:
+  requires:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    
   files:
     - test
   commands:
+  
     - conda inspect linkages -p $PREFIX qt  # [not win]
     - conda inspect objects -p $PREFIX qt  # [osx]
```

Comment:
We no longer need these lines but let's get this merged as soon as it passes. We can remove these with a `[skip ci]` PR later.